examples/asr/asr_adapters/conf/asr_adaptation.yaml

```diff
@@ -0,0 +1,114 @@
+
+name: "ASR-Adapter"
+
+model:
+  # One of the below two values must be set !
+  pretrained_model: null  # name of a pretrained model
+  nemo_model: null  # path to a ASR model file (.nemo)
+
+  adapter:
+    adapter_name: ???  # Name of the adapter, used by the script
+    _target_: nemo.collections.common.parts.adapter_modules.LinearAdapter
+    in_features: ???  # User must provide the output dimension of the layers of the model
+    dim: 32  # can be any hidden dimension as chosen by user, but small values are preferred to reduce param count
+    activation: swish
+    norm_position: 'post'  # Can be `pre` or `post`
+    # Optional global config available to all adapters
+    global_cfg:
+      encoder_adapter: True  # ASR adapter key, determines whether to add encoder adapter modules
```

Comment:
What happens if it is false? if it is false, does user still need to have the adapter section?

Reply:
If its false, currently adaptation will fail since we currently have only encoder adapters. 

This is built to have future support for adapters at any part of the model - preprocessor, encoder, decoder, joint, loss, etc. Anything and everything can be adapted after all, and we want to flexibly enable and disable which adapter we want to add with a given name.